### ISSUE TITLE
Create a landing page 

### ISSUE THREAD

**Issue by Ranjana993** [thecuriousteam/PyLibLog#39]
Create  a responsive landing  page with cool UI .

**Comment by Prakharpandey007** [comment]
Please assign me this issue. I will make sure that all of your points are fulfilled .
Thank you, looking forward for great response .

**Comment by thecuriousteam** [comment]
Yes You can Start working and make sure you are onboarded to community is not please join with the link given in readme.md
